lib/bike_brigade/importers/mailchimp_importer.ex

```diff
@@ -94,14 +94,14 @@ defmodule BikeBrigade.Importers.MailchimpImporter do
       We had trouble with the address for #{rider.name}. Please edit manually: #{BikeBrigadeWeb.Router.Helpers.rider_index_url(BikeBrigadeWeb.Endpoint, :edit, rider.id)}, and don't forget to remove the `invalid_location` tag!
     """
 
-    Task.start(SlackWebhook, :post_message, [error_message])
+    Task.start(Slack.Operations, :post_message, [error_message])
```

Comment:
I think that `:post_message` should be renamed to `:post_message!` in order to indicate that it can raise an exception (it calls `post!` which has the bang)

My thinking is that if we used `Task.start_link` here instead of `Task.start` the exception would crash the import.